Gemfile.lock

```diff
@@ -110,6 +110,7 @@ GEM
       thread_safe (~> 0.1)
     tzinfo-data (1.2018.5)
       tzinfo (>= 1.0.0)
+    wdm (0.1.1)
```

Comment:
Why the change in dependencies? Do we need these @knrt10 ?

Reply:
Please changes it back to what it was.